README.md

```diff
@@ -61,6 +61,47 @@ For more info about building packages refer to [rollup webpage](https://rollupjs
 </script>
 ```
 
+## Manually sending a message
+
+```html
+<div id="dreams-web-sdk-container"></div>
+
+<script src="path/to/dreams/sdk/js/file">
+  var callbacks = {
+    onIdTokenDidExpire: async () => {
+      const resp = await fetch("/token-expired-endpoint");
+      const data = await resp.json();
+
+      return data.token;
+    },
+    onAccountProvisionRequested: () => {
+      await fetch("/provision-account-enpoint")
+    },
+    onExitRequested: () => {
+      window.location.href = "http://example.com/some/path"
+    }
+  }
+
+  var sdk = new DreamsWebSDK("https://dreams.api.endpoint");
+  var messageHandler = sdk.setup(callbacks);
+
+  sdk.start(user_jwk_token_value, "en");
```

Comment:
do we need all of this setup to make the example of sending navigateTo? Could the example focus on the different message types instead?